build-logic/src/main/kotlin/floodgate-modded.platform-conventions.gradle.kts

```diff
@@ -133,6 +133,6 @@ modrinth {
     syncBodyFrom.set(rootProject.file("README.md").readText())
 
     uploadFile.set(tasks.remapJar.get().destinationDirectory.get().asFile.resolve("${versionName(project)}.jar"))
-    gameVersions.addAll(libs.minecraft.get().version as String, "1.21.4", "1.21.5")
+    gameVersions.addAll(libs.minecraft.get().version as String, "1.21.6")
```

Comment:
```suggestion
    gameVersions.add(libs.minecraft.get().version as String)
```
Otherwise we'll add 1.21.6 twice :p
The `libs.xyz...` references the Minecraft version defined gradle version library (libs.versions.toml), which is also 1.21.6